src/flow.h

```diff
@@ -499,6 +501,7 @@ typedef struct Flow_
     uint64_t todstbytecnt;
     uint64_t tosrcbytecnt;
 
+    bool elephant;
```

Comment:
how does this affect the flow memory layout? (pahole)

Reply:
This was creating a 7 byte hole. Moved it above where there was already a 2 byte hole which is now reduced to 1 byte